import sys

def CriticalError():
  sys.exit(5)
